draft-ietf-secevent-http-push.xml

```diff
@@ -171,7 +171,7 @@
             </t>
             <t>
                 The mechanisms by which the SET Recipient performs this validation
-                are out of scope for this document. SET parsing and issuer and
+                are out of scope for this document. SET parsing, issuer, and
                 audience identification are defined in <xref target="RFC8417"/>.
```

Comment:
It's "issuer identification" not "issuer" that is out of scope. The original text is grouped like this:

> (SET parsing) and (issuer and audience identification)

How about:
```suggestion
                are out of scope for this document. SET parsing and identification
                of issuers and audiences are defined in <xref target="RFC8417"/>.
```

Or:
```suggestion
                are out of scope for this document. SET parsing, issuer identification, and
                audience identification are defined in <xref target="RFC8417"/>.
```